.evg.yml

```diff
@@ -125,12 +240,12 @@ functions:
           set -o errexit
           
           mkdir -p $SBOM_TOOL_DIR
-          mkdir -p $SBOM_DIR
+          mkdir -p $SSDLC_DIR
```

Comment:
Once the directory is create in `export variables` then no need for this is anymore either